docs/shared/getting-started/intro.md

```diff
@@ -1,6 +1,32 @@
 # Intro to Nx
 
-Nx is a powerful open-source build system that provides tools and techniques for enhancing developer productivity, optimizing CI performance, and maintaining code quality. Learn more about [how Nx works](/getting-started/why-nx).
+Nx is a powerful open-source build system that provides tools and techniques for enhancing developer productivity, optimizing CI performance, and maintaining code quality. If you want to learn more about what Nx is, how it works and why you might want to use it, check out our ["Why Nx"](/getting-started/intro) page.
+
+If instead you want to jump right into it, run the following command. It will guide you through the setup:
+
+{% tabs %}
+{% tab label="npm" %}
+
+```{% command="npm create nx-workspace" path="~" %}
+
```

Comment:
```suggestion
```shell
npm create nx-workspace
```

Reply:
This looks better if you aren't going to show the output of the command

Reply:
I actually on purpose used the command because I felt it stands more out than just the normal box. Like one immediately sees this is a CLI command vs a code block etc. Otherwise everything looks the same.

We could adjust the style though by creating a custom `terminal` fence that does the same

Reply:
Like I personally find the 2nd better

![image](https://github.com/nrwl/nx/assets/542458/e0f46e22-33a7-44a4-a983-b1a5b8111796)

![image](https://github.com/nrwl/nx/assets/542458/b2b63034-ee62-4d6a-9131-08061fe5e474)

But we could check in the component whether there's an output and in that case reduce the padding below the command. + maybe allow writing it like

```
```terminal
npx create-nx-workspace ...
```

Reply:
I'm fine with the fake terminal window if you remove the padding on the bottom

Reply:
What do you think about just having a single terminal that uses NPM and ignoring the yarn / pnpm? We'd have that in the installation page anyway. It'd make it more concise and less loaded also